<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="GauravB-LT" name="com.test.SikuliMaven.PkiToolApplicationExitTest" tests="1" failures="0" timestamp="14 Jun 2017 12:57:06 GMT" time="0.255" errors="1">
  <testcase name="pkiToolApplicationExit" time="0.255" classname="com.test.SikuliMaven.PkiToolApplicationExitTest">
    <error type="java.lang.ThreadDeath">
      <![CDATA[java.lang.ThreadDeath
	at java.lang.Thread.stop(Unknown Source)
	at org.sikuli.script.RunTime.abortScripting(RunTime.java:73)
	at org.sikuli.script.Image.getImageFromTarget(Image.java:614)
	at org.sikuli.script.Region$RepeatableFind.<init>(Region.java:3129)
	at org.sikuli.script.Region.wait(Region.java:2721)
	at org.sikuli.script.Region.find(Region.java:2336)
	at com.test.SikuliMaven.PkiToolApplicationExit.pkiToolApplicationExit(PkiToolApplicationExit.java:15)
	at com.test.SikuliMaven.PkiToolApplicationExitTest.pkiToolApplicationExit(PkiToolApplicationExitTest.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- pkiToolApplicationExit -->
</testsuite> <!-- com.test.SikuliMaven.PkiToolApplicationExitTest -->
